enterprise/api/v1/views.py

```diff
@@ -455,6 +461,36 @@ class EnterpriseCustomerReportingConfigurationViewSet(EnterpriseReadOnlyModelVie
     filter_fields = FIELDS
     ordering_fields = FIELDS
 
+    @permission_required(
+        'enterprise.can_manage_reporting_config',
+        fn=lambda request, *args, **kwargs: get_ent_cust_from_report_config_uuid(kwargs['uuid']))
+    def retrieve(self, request, *args, **kwargs):
```

Comment:
should we consider adding some sort of permission requirement for list? it seems weird that you could see the same data without having the role by hitting the list endpoint but be restricted when you hit retrieve...